config/config/common-config.yaml

```diff
@@ -0,0 +1,358 @@
+## Common Kubernetes configuration options.
```

Comment:
Does this file contain only the common fields between sc and wc configs?

Reply:
Yes, it contains all fields where the values was the same between sc and wc configs.